bindings/GeometryDash.bro

```diff
@@ -5,6 +5,7 @@ class GDString {
     GDString& winAssign(GDString const&, size_t, size_t) = win 0xf720;
     GDString& winAssign(char const*) = win 0xf680;
     GDString& winAssign(char const*, size_t) = win 0xf840;
+	GDString& operatorPlus(char const*, size_t) = win 0x156F0;
```

Comment:
not sure why this was added since its just an unused internal function but you go girl